README.md

```diff
@@ -2,16 +2,16 @@
 
 NAT Gateways are dead. Long live NAT instances!
 
-Built and released with 💚 by <a href="https://chime.com"><img src="/assets/Chime_company_logo.png" alt="Chime Engineering" width="146"/></a>
+Built and released with 💚 by <a href="https://chime.com"><img src="/assets/Chime_company_logo.png" alt="Chime Engineering" width="60"/></a>
```

Comment:
I made this smaller because I thought it looked cleaner but if you don't like I can go back to what it was before.
<img width="266" alt="Screenshot 2024-05-09 at 10 13 17 AM" src="https://github.com/1debit/alternat/assets/6735054/88ed6b8f-681c-463e-abfd-14de9247f437">

Reply:
LGTM!